docs/release-notes/generate-release-note.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+categories=("Bug Fixes" "Security Fixes" "Breaking Changes" "Improvements" "New Features")
+components=("WebUI" "Notebook" "TensorBoard" "Command" "Shell" "Experiment" "API" "Images")
+
+autocomplete() {
+    local items=("$@")
+    select item in "${items[@]}"; do
+        if [[ -n $item ]]; then
+            echo "$item"
+            break
+        else
+            echo "Invalid selection"
+        fi
+    done
+}
+
+echo "Enter the filename for the release note (without extension):"
+read filename
+
+echo "Choose a category (or enter a new one):"
+category=$(autocomplete "${categories[@]}")
+
+echo ":orphan:" >"${filename}.rst"
```

Comment:
Maybe always put the file in the right directory (roughly `dirname $0`). Might also be nice to have a variable for the full output path.